source/guide_ackee.rst

```diff
@@ -34,47 +32,81 @@ Ackee_ is released under the `MIT License`_.
 Prerequisites
 =============
 
-Ackee needs a working :lab:`MongoDB <guide_mongodb>` installation. 
-Follow and install MongoDB as written in the Uberlab guide :lab:`here <guide_mongodb>`.
+Ackee_ requires a MongoDB_ database. Set it up using the :lab:`MongoDB Uberlab guide <guide_mongodb>`. You will need your MongoDB_ admin name and password later.
 
-Then setup your domain:
+Set up your domain:
 
 .. include:: includes/web-domain-list.rst
 
-Also install ``yarn`` globally to later use as package manager for :manual:`Node.js <lang-nodejs>`.
+Installation
+============
+
+Install ``yarn`` globally:
 
 .. code-block:: console
 
   [isabell@stardust ~]$ npm install yarn -g
-  /home/isabell/bin/yarn -> /home/isabell/lib/node_modules/yarn/bin/yarn.js
-  /home/isabell/bin/yarnpkg -> /home/isabell/lib/node_modules/yarn/bin/yarn.js
-  + yarn@1.22.4
-  installed 1 package in 1.002s
   [isabell@stardust ~]$ 
 
-Installation
-============
+Clone the `GitHub repository <https://github.com/electerious/Ackee>`_ and install the dependencies:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ git clone https://github.com/electerious/Ackee
+  [isabell@stardust ~]$ cd Ackee
+  [isabell@stardust Ackee]$ yarn
+
+Configuration
+=============
+
+Ackee Config
+------------
 
-Clone the `GitHub repository of Ackee`_:
+Create a new, empty ``.env`` file in the root of the Ackee_ directory. Add the following content, replacing the placeholders. The environment variables ``ACKEE_USERNAME`` and ``ACKEE_PASSWORD`` define the credentials to log into the web interface. If you want to use Ackee_ to analyze a domain other than ``isabell.uber.space``, you have to add it under ``ACKEE_ALLOW_ORIGIN`` (read more about the CORS headers `in the Ackee documentation <https://github.com/electerious/Ackee/blob/master/docs/CORS%20headers.md>`_). 
```

Comment:
You always need to add a CORS header for every site you want to use the tracking script. No matter if it's your standard Uberspace domain or not.

Reply:
Are you sure? Shouldn't it be possible to analyze the own host? E.g. test.isabell.uber.space?

Reply:
No, that's not possible. You'd need to specify some kind of wildcard for that.

Reply:
I updated the guide accordingly. It should make more sense now. I added a note for multple domains.